arangodb-net-standard/DocumentApi/DocumentApiClient.cs

```diff
@@ -181,6 +197,33 @@ public async Task<T> GetDocumentAsync<T>(string documentId)
             throw await GetApiErrorException(response);
         }
 
+        /// <summary>
+        /// Get multiple documents
+        /// </summary>
+        /// <typeparam name="T"></typeparam>
+        /// <param name="collectionName"></param>
+        /// <param name="selectors"></param>
+        /// <returns></returns>
+
+        public async Task<List<T>> GetDocumentsAsync<T> (string collectionName, IList<string> selectors)
+        {
+            string uri = $"{_docApiPath}/{WebUtility.UrlEncode(collectionName)}?onlyget=true";
+
+            var content = GetContent(selectors, false, true);
```

Comment:
I think it makes sense to ignore null values here, I noticed that we don't do it in other methods like `DeleteDocumentsAsync`. I think we can create another issue to do it there. @rossmills99 any thoughts?

Reply:
Do you mean to ignore null values in the `selectors` list?  I'm not sure if the serializer would support doing that for a list type.  Have you tried?  Alternatively we could filter selectors if this was a needed feature, e.g. `selectors = selectors.Where(s => s != null);`.

Reply:
I did a test using C# interactive in Visual Studio:
```
> var jsonSettings = new JsonSerializerSettings
. {
.     NullValueHandling = NullValueHandling.Ignore
. };
> string json = JsonConvert.SerializeObject(new List<string> { "test", null, "test2" });
> json
"[\"test\",null,\"test2\"]"
```
As you can see, using `NullValueHandling.Ignore` has no effect on the List values, I think it is only used for property/field values.